python/ray/data/_internal/lazy_block_list.py

```diff
@@ -544,9 +544,9 @@ def _submit_task(
         """Submit the task with index task_idx.
 
         NOTE: When dynamic block splitting is enabled, returns
-        Tuple[ObjectRef[ObjectRefGenerator], None] instead of
+        Tuple[ObjectRef[DynamicObjectRefGenerator], None] instead of
```

Comment:
cc @raulchen is this still relevant with data?

Reply:
this code path is being deprecated.